Sources/KlaviyoSwift/Klaviyo.swift

```diff
@@ -121,9 +138,10 @@ public class Klaviyo: NSObject  {
      
      - Parameter userInfo: NSDictionary containing the push notification text & metadata
      */
+    @objc
     public func handlePush(userInfo: NSDictionary) {
-        if let body = userInfo["body"] as? NSDictionary, let _ = body["_k"] {
-            trackEvent(eventName: KLPersonOpenedPush, properties: userInfo)
+        Klaviyo.sdkInstance.handle(remoteNotification: userInfo as! [AnyHashable : Any]) { result in
+            // Empty implementation
```

Comment:
Is this supposed to be empty? In the prior version didn't this method trigger `$opened_push`

Reply:
+1 this caught my eye too

Reply:
I'm calling into my own API. Our old SDK api doesn't take in a completion handler. We will have to intro a new objc api to take a completion handler.